tools/gen_struct_info.py

```diff
@@ -141,6 +141,8 @@ def parse_c_output(lines):
 
   for line in lines:
     arg = line[1:].strip()
+    if '::' in arg:
+      arg = arg.split('::', 1)[1]
```

Comment:
I'm surprised this works... is the namespace used in the file, I guess, so it's optional?

Reply:
This just allows us to write `C_STRUCTS.__cxa_exception` in the JS library code instead of `C_STRUCTS.__cxxabiv1::__cxa_exception`..  as long as there are no collisions it works fine.